lib/db/project.ts

```diff
@@ -83,5 +86,6 @@ export async function deleteProject(id: number): Promise<Project> {
       id: id,
     },
   });
+  createActivityLog("Delete", "project", project.id);
```

Comment:
One thing I didn't consider: once you delete a record (like the project here), you can't refer to that `project_id` anymore. The solution is probably to do a soft delete (?) but we'll just skip creating activity logs for delete events.